cognite/client/data_classes/aggregations.py

```diff
@@ -1,9 +1,12 @@
 from __future__ import annotations
 
-from abc import ABC
+from abc import ABC, abstractmethod
 from dataclasses import dataclass, field
-from typing import Any, ClassVar, Type, TypeVar, cast, final
+from typing import Any, ClassVar, Type, TypeVar, Union, cast, final
 
+from typing_extensions import TypeAlias
```

Comment:
You need to `poetry add` this as a dependency. I'm for it, wdyt @erlendvollset ?

Reply:
Nice, pickup. I did not notice I got this through the `mypy` installation. Agreed, would be great to have the 3.10 type hints backported